packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -939,10 +950,18 @@ export async function cleanup(compass?: Compass): Promise<void> {
       }
     }
     clearTimeout(timeoutId);
-    return;
+    return 'closed';
   })();
 
-  return Promise.race([timeoutPromise, closePromise]);
+  const result = await Promise.race([timeoutPromise, closePromise]);
+  if (result === 'timeout' && compass.mainProcessPid) {
+    try {
+      debug(`Trying to manually kill Compass [${compass.name}]`);
+      tryKillProcess(compass.mainProcessPid, compass.name);
+    } catch {
+      /* already logged ... */
+    }
```

Comment:
This is the most significant change to the tests, as discussed in standup yesterday, just killing the process if it didn't exit on its own seems to be the most viable option right now